Assets/Scripts/Meteor/MeteorSpawn.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MeteorSpawn : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject m_MeteorPrefab;
+    [SerializeField]
+    private Transform m_StartPointTransform;
+    [SerializeField]
+    private Transform m_EndPointTransform;
+
+    private Vector3 m_StartPoint;
+    private Vector3 m_EndPoint;
+
+    private void Start()
+    {
+        m_StartPoint = m_StartPointTransform.position;
+        GameObject meteor = Instantiate(m_MeteorPrefab, m_StartPoint, Quaternion.identity);
+
+        m_EndPoint = m_EndPointTransform.position;
+
+        RotateTo(meteor, m_EndPoint);
+    }
```

Comment:
This script spawns a meteor object on `Start`. But something has to spawn this script in the first place. Why can't that same place spawn the meteor and then we can ditch this script altogether?